tests/implementor/test_impl_schedule_blocked_fault.py

```diff
@@ -0,0 +1,74 @@
+from uuid import UUID
+
+from src import implementor as impl
+from src.fault_injector.fault_configurations.schedule_blocked_fault_configuration import (
+    ScheduleBlockedFaultConfiguration,
+    ScheduleBlockedFaultConfigurationXSimulationConfiguration,
+)
+
+
+class TestScheduleBlockedFaultConfiguration:
+    def test_get_all_schedule_blocked_fault_configuration_ids(
+        self, token, schedule_blocked_fault_configuration_data
+    ):
+        config = ScheduleBlockedFaultConfiguration.create(
+            **schedule_blocked_fault_configuration_data
+        )
+
+        response = impl.component.get_all_schedule_blocked_fault_configuration_ids(
+            {}, token
+        )
+        (result, status) = response
+        assert status == 200
+        assert str(config.id) in result
```

Comment:
Maybe you could check if `config.id` is at the right position in `response.json` instead of just checking if it is somewhere in the string :)